README.md

```diff
@@ -13,37 +15,50 @@ The target of ISCN is to build a general digital content registration schema, an
+- [Record Parent IPLD](schema/recordParentIPLD/README.md): The IPLD hash of the previous version of this record.
+- [Reward Proportion](schema/rewardProportion/README.md): The suggested reward propotion of a stakeholder.
+- [Content Fingerprints](schema/contentFingerprints/README.md): Fingerprints of the content, e.g. cryptographic hashe, URI, etc.
+- [Entity](schema/entity/README.md): Entity who should be cited and / or rewarded for the contribution on the content.
+- [Record Timestamp](schema/recordTimestamp/README.md): The timestamp of the current version of the ISCN record.
+- [Stakeholders](schema/stakeholders/README.md): List of StakeholderInfo contributed to this content.
+
+A full list of schema can be found at [schema](https://github.com/likecoin/iscn-specs/tree/master/schema), and ISCN record samples can be found in [here](https://github.com/likecoin/iscn-specs/tree/develop/sample).
 
 ## Linked Data
+
 Linked data is structured data that can be looked up via some methods, HTTP, RDF and URI but not limited to are all accepted.
 
 ## ISCN Architecture
-The complete ISCN registration is consist of a kernel [ISCN](schema/iscn/v1.md) core metadata to record the unique global identifier and maintain the stakeholders, the intellectual property rights and the content metadata by [Stakeholder](schema/stakeholder/v1.md), [Rights](schema/rights/v1.md) and [Content](schema/content/v1.md) corresponsibly. The architecture of an ISCN is shown below:
+
+The complete ISCN registration is consist of a kernel [ISCN](schema/iscn.md) record to record the unique global identifier and maintain the stakeholders, and the content metadata by [stakeholders](schema/stakeholders/README.md), [contentFingerprints](schema/contentFingerprints/README.md) and [contentMetadata](schema/contentMetadata/README.md) corresponsibly. Other metadata can be added flexiblely through [JSON-LD](https://json-ld.org/) and [schema.org](https://schema.org/) standard. The basic architecture of an ISCN is shown below:
 
 ![ISCN architecture](https://gateway.pinata.cloud/ipfs/QmT3gD8KvowzpaU2n4ppc7EHnJ7rgqkbRCqgKmdvYHSTiv)
 
 ## ISCN content registry
-A registry is a service provider for the ISCN registration, and a registry should provide service to register a digital content with metadata that follows the ISCN specification and to query. If any entities want to become a registry for ISCN, they should reserve a code for their registry in [here](https://github.com/likecoin/iscn-registry).
+
+A registry is a service provider for the ISCN registration, and a registry should provide service to register a digital content with metadata that follows the ISCN specification and to query. If any entities want to become a registry for ISCN, they should reserve a code for their registry in [here](https://github.com/likecoin/iscn-registry-index).
```

Comment:
Code reserve is not needed anymore, since we use registry name directly instead of numeric code.
May instead encourage potential registries to see if their name are already used, and to leave their registry name and related info (e.g.  description,URL to explorer) in the repo